tsdb/compact_test.go

```diff
@@ -786,6 +809,57 @@ func TestCompaction_populateBlock(t *testing.T) {
 	}
 }
 
+func GetOversizedChunk() [][]sample {
```

Comment:
I would just inline those with proper comment and definitely make it private.

Reply:
Sure. Thanks for the review.

Reply:
Hey bwplotka can you please clarify by what you mean when you say inline?
Do you mean something like this? `for i := 0; i < 200; i++ { chunkOne[i] = sample{t: int64(i + 1)}}`. This wont go through because gofmt will change the style anyways...

Reply:
I meant that I would suggest copying the whole thing to the place you use it. It can be multiline, but having a separate function just for one usage is anti-pattern.

You can use anonymous functions `func() { ... }`